source/common/upstream/thread_aware_lb_impl.cc

```diff
@@ -30,7 +30,7 @@ void ThreadAwareLoadBalancerBase::refresh() {
   }
 
   {
-    std::unique_lock<std::shared_timed_mutex> lock(factory_->mutex_);
```

Comment:
can we also have a check_format test for std::shared_time_mutex and std::unique_lock?